app/views/govuk_publishing_components/components/docs/input.yml

```diff
@@ -119,17 +119,20 @@ examples:
       name: "name"
       width: 10
   with_search_icon:
+    description: |
+        Adds a search icon, `spellcheck="true"` is also present when the value is passed.
```

Comment:
A very finicky comment (apologies!) but just wonder if this could be misinterpreted that spellcheck is added when `search_icon: true` is present.

Perhaps should word slightly differently. Something like...

```
Adds a search icon.

[spellcheck](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/spellcheck) can also be added to indicate that the element should be, if possible, checked for spelling errors
```

Reply:
Works for me. ✅